scripts/travis-tests

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env node
+
+const { execFileSync, spawnSync } = require('child_process')
+
+const run = (command, args) => {
+  const { status } = spawnSync(command, args, { stdio: 'inherit' })
+  if (status !== 0) {
+    process.exit(status)
+  }
+}
+
+const getFiles = () =>
+  execFileSync(
+    'git',
+    [
+      'diff',
+      '--diff-filter=AM',
+      '--ignore-submodules',
+      '--name-only',
+      'master',
+    ],
+    { encoding: 'utf8' }
+  )
+    .split('\n')
+    .filter(_ => _ !== '')
+
+// -----------------------------------------------------------------------------
+
+if (process.env.TRAVIS_PULL_REQUEST) {
```

Comment:
I don't think this test works, AFAIU, `TRAVIS_PULL_REQUEST` contains the string `false` if the current job is not a PR, and all non-empty strings are truthy in JS.

Reply:
How come our previous tests did not show this issue?

How can we be sure it's now properly working?

Reply:
Because to simulate master, I simply reversed the condition.

But to be sure, you'll have to test a merge on master. Do you have any other ideas?

Reply:
Fine by me, after merging we'll take a look.